package.json

```diff
@@ -2,7 +2,7 @@
     "name": "berry-material-react-free",
```

Comment:
now that we're at it could you also change the name on this line to something like 'smart-ledger', since this file defines the name of our npm project.